manifests/s/SimonG4ELI/WorldMap/1.4.0/SimonG4ELI.WorldMap.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: SimonG4ELI.WorldMap
+PackageVersion: 1.4.0
+InstallerType: nullsoft
+Installers:
+- InstallerUrl: https://www.dropbox.com/scl/fi/k8nzcys5n8pe1sc7zu7tm/Simon-s-World-Map-2024-02-04-19h24m.exe?rlkey=abskz0i1tyjgsbwlcw5pufxti&dl=1
+  Architecture: x86
+  InstallerSha256: C14CE4B64A2A374B96D207DA4F25B3968F11BBB753F72C5E4AAD9838C96758B4

```

```suggestion
  InstallerSha256: C14CE4B64A2A374B96D207DA4F25B3968F11BBB753F72C5E4AAD9838C96758B4
  AppsAndFeaturesEntries:
    - DisplayVersion: 2024-02-04 19h24m
```
Required to match the package version for upgrade